### PR TITLE
VCABS-79 Customize Pipeline Payload

### PR DESCRIPTION
**Pull request by SnitavetsIV**

### CODE REVIEW

File: backend/backend/handlers/workflows/executeWorkflow.py [hunk 1]
```diff
@@ -35,12 +34,27 @@ def get_pipelines(databaseId, pipelineId):
     )
     return response['Items']
 
-def launchWorkflow(bucketName, key, workflow_arn, asset_id, workflow_id, database_id):
+def launchWorkflow(asset, bucketName, key, workflow_arn, workflow_id, database_id):
     print("Launching workflow with arn: ", workflow_arn)
+    asset_id = asset['assetId']
+    metadata = get_metadata(database_id, asset_id)[0]
+    print(metadata)
```
Comment (kcopsguru) [inline backend/backend/handlers/workflows/executeWorkflow.py]: Is this for debugging or real logs? if for real logs, let's use descriptive log messages.

File: infra/lib/api-builder.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ import { buildMetadataSchemaService } from "./lambdaBuilder/metadataSchemaFuncti
 import { buildMetadataFunctions } from "./lambdaBuilder/metadataFunctions";
 import { buildUploadAssetWorkflow } from "./uploadAssetWorkflowBuilder";
 import { buildAuthFunctions } from "./lambdaBuilder/authFunctions";
+import {EnvProps} from "./infra-stack";
```
Comment (kcopsguru) [inline infra/lib/api-builder.ts]: code format is different here. does VAMS have linting rules?

**Review by kcopsguru — APPROVED** [review]
LGTM. just some minor issues.

File: infra/lib/api-builder.ts (hunk 2 above)
Comment (SnitavetsIV) [reply infra/lib/api-builder.ts]: VAMS doesn't have linting, format is everywhere is different, but I will fix this case

**Pull request merged**